### ISSUE TITLE
Intellisense

### ISSUE THREAD

**Issue by ofirmoss** [Microsoft/vscode#58510]
Issue Type: <b>Bug</b>

after a few minutes the Intellisense stops working, even after restarting the visual studio code

VS Code version: Code 1.27.1 (5944e81f3c46a3938a82c701f96d7a59b074cfdc, 2018-09-06T09:21:18.328Z)
OS version: Windows_NT x64 10.0.17134

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7200U CPU @ 2.50GHz (4 x 2712)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|7.88GB (1.60GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (12)</summary>

Extension|Author (truncated)|Version
---|---|---
path-intellisense|chr|1.4.2
vscode-togglecodelens|cml|1.0.3
gitlens|eam|8.5.6
EditorConfig|Edi|0.12.4
tslint|eg2|1.0.39
auto-rename-tag|for|0.0.15
Angular-BeastCode|Mik|6.2.23
debugger-for-chrome|msj|4.10.1
vetur|oct|0.12.6
vscode-html-scss|P-d|0.0.42
vscode-icons|rob|7.26.0
html-css-class-completion|Zig|1.17.1

</details>

**Comment by programmingj** [comment]
I have the exact same issue on Windows 10.  I can't "Find all references", "Go to definition"/"Peek definition" and no intellisense on anything.  I also can't go back to 1.26.1 User version because even after uninstalling 1.27.1 and deleting the setting folder (didn't know you had to go to a different folder to delete extensions, I do now) then installing 1.26.1 User version I still had no intellisense or the other features and it immediately installed 1.27.1 again once I closed Code.

I am going to have to try installing the Admin version because I CAN NOT have my dev environment hosed for days just because of forced updates and issues like this.

**Comment by joaomoreno** [comment]
@ofirmoss What language are you authoring?

**Comment by programmingj** [comment]
C# and TypeScript (Angular) mostly. But we found the problem with this:
someone checked in a sln file for an SSIS project in a subfolder and it was
stopping OmniSharp in weird ways.

   However, now that I have installed and reinstalled Code so many times
and so many versions, then backed out OmniSharp to the previous version,
then upgraded it back to the current version (using the Update button in
the extension) i no longer can get a terminal window!!! That also means no
logs, Output window, etc. I have uninstalled everything, re-booted my PC,
checked the path environment variable as well as ensured Code is pointing
to the correct folder for CMD and tried Power Shell all to no avail.

   I am completely stopped in my tracks until i can get Code to open a
terminal now so it can run dotnet CLI commands!

**Comment by programmingj** [comment]
I have traced my terminal issue down to winpty but I can't find the
workaround for it that is stated exists on the support entry for it. Anyone
know how to get the terminal window problems to go away??

**Comment by martellaj** [comment]
I'm also not seeing Intellisense work for TypeScript (Insiders).

**Comment by programmingj** [comment]
Make sure you look in your log entries for things like errant sln files,
etc. But don't uninstall and reinstall prior versions before you know for
sure you don't have something like we had! I STILL have a non-workinh Code
environment because I can no longer get a working terminal after installing
every possible combination of versions of tools, every order of
installation, upgrading, not upgrading, renaming every winpty.dll
occurrence I could find, the list goes on...

**Comment by joaomoreno** [comment]
@programmingj Please do not hijack other people's issues. Please create a separate issue for your terminal problems. You can cc me there.

@ofirmoss Still waiting on your reply.

**Comment by keithoconor** [comment]
People in the office here has been having the same issue. I'm on Windows 10, VSCode 1.27.2, editing C# with OmniSharp plugin 1.16.1, opening solutions generated by Unity 2018.2. It's been happening since at least 1.27.1, and maybe 1.27. 

It works initially, but stops working after a short amount of time. There doesn't seem to be any reliable repro steps, just generally using Ctrl+click on symbols and switching between files seems to be enough to do it.

**Comment by keithoconor** [comment]
@joaomoreno If you need any other information, please let me know. The team has stopped using VS Code and are currently migrating to either Visual Studio or Jetbrains Rider, since being without intellisense is not workable.

**Comment by joaomoreno** [comment]
@keithoconor Please do not hijack other people's issues. Please create a separate issue for your Intellisense problems. Since that is C# specific, you might wanna create an issue on that extension: https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp

@ofirmoss Still waiting on your reply.
